.openpublishing.redirection.json

```diff
@@ -21,7 +21,7 @@
       "redirect_document_id": true
     },
     {
-      "source_path": "docs/supported-platforms-visual-cpp.md.md",
```

Comment:
The extra `.md` causes the redirection to not work (same for the other occurrence).